EIPS/eip-5792.md

```diff
@@ -122,38 +163,66 @@ Returns the status of a call batch that was sent via `wallet_sendCalls`. The ide
 type GetCallsParams = string;
 
 type GetCallsResult = {
-  status: 'PENDING' | 'CONFIRMED';
+  version: string;
+  id: `0x${string}`;
+  chainId: `0x${string}`;
+  status: number; // See "Status Codes"
   receipts?: {
     logs: {
       address: `0x${string}`;
       data: `0x${string}`;
       topics: `0x${string}`[];
     }[];
     status: `0x${string}`; // Hex 1 or 0 for success or failure, respectively
-    chainId: `0x${string}`;
     blockHash: `0x${string}`;
     blockNumber: `0x${string}`;
     gasUsed: `0x${string}`;
     transactionHash: `0x${string}`;
   }[];
+  capabilities?: Record<string, any>;
```

Comment:
what happens if these caps changed since last interaction? i'm curious what these are doing here.  just for matching?

Reply:
> L153: The `capabilities` object allows the wallets to attach a capability-specific metadata to the response.

Meaning, this field contains any data specific to a requested "capability", not the list of supported capabilites as in `wallet_getCapabilities`.